### PR TITLE
Fix for #126

### PR DESCRIPTION
**Pull request by mirkosertic**

### CODE REVIEW

File: src/main/java/pl/touk/sputnik/processor/sonar/SonarResultParser.java [hunk 1]
```diff
@@ -118,9 +118,20 @@ private String getIssueFilePath(String issueComponent, Map<String, Component> co
...
         if (!Strings.isNullOrEmpty(comp.moduleKey)) {
-            Component moduleComp = components.get(comp.moduleKey);
-            if (!Strings.isNullOrEmpty(moduleComp.path)) {
-                file = moduleComp.path + '/' + file;
+            String theKey = comp.moduleKey;
+            while (!theKey.isEmpty()) {
+                Component theChildComp = components.get(theKey);
+
+                int p = theKey.lastIndexOf(":");
+                if (p>0) {
```
Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/processor/sonar/SonarResultParser.java]: [Checkstyle] ERROR: '>' is not preceded with whitespace.

Comment (sputnikci) [inline src/main/java/pl/touk/sputnik/processor/sonar/SonarResultParser.java]: [Checkstyle] ERROR: '>' is not followed by whitespace.

Comment (pjagielski) [inline src/main/java/pl/touk/sputnik/processor/sonar/SonarResultParser.java]: +1, can you fix just that and we'll merge

**Comment by pjagielski** [comment]
thanks!

**Comment by SpOOnman** [comment]
Oh, I didn't see it was merged already.

@mirkosertic can you provide an unit test for this change? Thanks.

**Pull request merged**